docs/development/styles.md

```diff
@@ -0,0 +1,89 @@
+# Styles
+
+## Recommended Components
+
+N/A
+
+## Guidance
+
+During our discussions it became clear that there are multiple ways of styling and that there is a lot of choice, but there was one common idea that kept recurring.  If you are using a CSS framework, be sure to use what that framework is using.
+
+### CSS Frameworks
+
+CSS frameworks are a great choice for developing applications.  Some advantages are:
+
+* Great for out-of-the-box prototyping
+* customizable
+* consistent
+
+[Tailwind CSS](https://tailwindcss.com/) is a modern choice that the team has used.  Other recommendations that the team made are [Carbon](https://carbondesignsystem.com/) and [Patternfly](https://www.patternfly.org/) which are both IBM and Red Hat projects respectively.
+
+It is also important that if you are using a front-end JavaScipt framework, the CSS framework should fit into it.
+
+### Layout Methods
+
+Since it is generally harder to design for a smaller screen, the team recommends a mobile first and responsive approach to an applications layout.  When defining css rules it is recommended to put the rules for mobile first followed by the rules for other layouts that your application might have.
+
+The use of [Media Queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries) allows you to apply CSS styles to your application using various characteristics, such as screen resolution and orientation.  Media Queries will help with the responsive approach to writing applications as mentioned above.
+
+Another way of making an application responsive is with either the Grid or Flex layouts.  The team is familiar with both Grid and Flex, while Grid is slightly prefered.  Depending on the situation, both can be used.
+
+Whichever layout method is used, Grid or Flex, will go hand in hand with using Media Queries
+
+Of course, if you are using a CSS framework it is important to use whichever layout method is built in there.
+
+### Preprocessors
+
+While preprocessors are still being used, built-in css properties and rules are now starting to deprecate the use of preprocessors.  It is also important to decide if the scope and scale of your project needs one.
+
+However, the team does prefer the use of the [Sass](https://sass-lang.com/).
+
+The team also agreed that it was important to be more CSS centric(native CSS features) even when using a preprocessor.  This was because the language is introducing more and more features, which will start to make these preprocessors deprecated.
```

Comment:
```suggestion
The team also agreed that it was important to be more CSS-centric (maximize use of native CSS features) even when using a preprocessor.  This was because the language is introducing more and more features, which will start to make these preprocessors deprecated.
```